recipes/gpboost/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gpboost" %}
+{% set posix = 'm2-' if win else '' %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 617154763063b2c0e45d24b609e3877b91ca60e4c40d93439280b2ec35b6eeec
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}make
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - setuptools
+  host:
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scikit-learn !=0.22.0
+    - scipy
+    - wheel
+  run:
+    - numpy
+    - pandas
+    - python
+    - scikit-learn !=0.22.0
+    - scipy
+    - wheel
+
+test:
+  imports:
+    - gpboost
+
+about:
+  home: "https://gpboost.readthedocs.io/en/latest/"
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
Please include the licenses for all of the libraries this library is static-linking / revending.

https://github.com/fabsig/GPBoost/tree/master/external_libs

Reply:
I'll have another look at this next week. I would hope it is not vendoring all of those. I'll check. If it is I'll try to patch the build system to at least get it to build against a conda-forge packaged eigen as the licensing situation for that one is entangled with the build system and so might be tricky to navigate by directly trying to including the license.